apis/slack/tool.gpt

```diff
@@ -19,6 +19,7 @@ Name: Get Channel History
 Description: Get the chat history for a channel in the Slack workspace
 Share Context: Slack Context
 Credential: github.com/gptscript-ai/gateway-oauth2 as slack.write with SLACK_TOKEN as env and slack as integration and "channels:history groups:history im:history mpim:history channels:read files:read im:read search:read team:read users:read groups:read chat:write groups:write mpim:write im:write" as scope
+Share Tools: List Channels
```

Comment:
In each of the places where we share `List Channels`, can we also share `Search Channels`? The latter is a lot more useful for Slack workspaces that have a ton of channels.